README.md

```diff
@@ -79,7 +79,7 @@ export PATH=$GOBIN:$PATH
 With this method, you generate the binary manually (need Go installed), downloading and compiling the source code:
 
 ```bash
-git clone https://github.com/Layr-Labs/eigenlayer-cli.git
+git clone https://github.com/Layr-Labs/eigenlayer-cli.git && mv eigenlayer-cli eigenlayer
```

Comment:
actually better would be to change `cd eigenlayer` to `cd eigenlayer-cli` in next line @neuweltgeld

Reply:
if ``eigenlayer-cli`` is to be used as folder name, PATH variable also be changed
````
# Using Go
sudo cp $GOPATH/bin/eigenlayer /usr/local/bin/

# Build from source
sudo cp eigenlayer/build/eigenlayer /usr/local/bin/
````

Reply:
yea - good catch - feel free to change that too. It's better than renaming the repo.

Reply:
> yea - good catch - feel free to change that too. It's better than renaming the repo.

done √

Reply:
Thank you!